scss/_orange-navbar.scss

```diff
@@ -60,3 +59,99 @@
     }
   }
 }
+
+.global {
```

Comment:
I'm not sure we need this `.global` class. The component for the headers already exists in https://boosted.orange.com/docs/5.1/components/navbar/. We should modify it without adding a new `.global` class with "Boosted mod" like it was done in v4.
Otherwise, Orange developers won't understand the difference between Bootstrap-half-modified-navbars and Navbars with the `.global` class I think.